content/docs/atom-guide/dev-guide/troubleshoot.md

```diff
@@ -63,4 +63,27 @@ You can enable validation in your RHI device to output additional debugging info
 
 - `--rhi-device-validation="enable"`: Enables basic driver validation.
 - `--rhi-device-validation="verbose"`: Enables basic driver validation and additional reporting.
-- `--rhi-device-validation="gpu"`: Enables GPU-based validation (GBV). This validation mode runs slower, but catches issues that the `enable` and `verbose` modes may not catch. Note that this mode may produce false positives.
\ No newline at end of file
+- `--rhi-device-validation="gpu"`: Enables GPU-based validation (GBV). This validation mode runs slower, but catches issues that the `enable` and `verbose` modes may not catch. Note that this mode may produce false positives.
+
+
+## DRED logs (Windows DirectX Only)
+
+When a TDR (Timeout Delay Recovery) crash occurs while using the DX12 backend specifically, [Device Removed Extended Data](https://microsoft.github.io/DirectX-Specs/d3d/DeviceRemovedExtendedData.html) (aka "DRED") breadcrumbs
+are written to a timestamped log file in `.o3de/Logs/DRED/`. This log contains all events, draws, barriers, and dispatches leading up to a crash, as well as commands that failed to complete.
+The offending command that likely caused the device removal will be visible after the `===ERROR START===` and before the `===ERROR END===` lines in the log. 
```

Comment:
I was curious and looked up Timeout Delay Recovery for DX12, and saw "Timeout Detection and Recovery", which can cause TDR delay crashes. If that's accurate, I suggest changing to "Timeout Detection and Recovery (TDR) delay crash". 

```suggestion
When a Timeout Delay Recovery (TDR) crash occurs while using the DirectX12 backend, [Device Removed Extended Data](https://microsoft.github.io/DirectX-Specs/d3d/DeviceRemovedExtendedData.html) (DRED) breadcrumbs are written to a timestamped log file in `.o3de/Logs/DRED/`. This log contains all of the events, draws, barriers, and dispatches that led up to a crash, as well as commands that failed to complete. The offending command that likely caused the device removal is visible after the `===ERROR START===` and before the `===ERROR END===` lines in the log. 
```

Reply:
Nope this was purely a typo thanks for the catch! Will fix to just "Timeout Detection and Recovery"